starter_project_backend_nest/src/article/article.service.ts

```diff
@@ -36,9 +36,9 @@ export class ArticleService {
 
   async deleteArticleById(id: string) {
     try {
-      await this.getArticleById(id);
+      let article = await this.getArticleById(id);
       await this.articleModel.findByIdAndDelete(id);
-      return `success`;
+      return article;
```

Comment:
after deleting the article you don't need to return the article object that you deleted